.build/build.ps1

```diff
@@ -61,46 +53,42 @@ task Clean {
     }
 }
 
+task Compile {
+    Write-Info "Running 'dotnet build'"
 
-# RestorePackages task, restores all the NuGet packages.
-task RestorePackages {
-    Write-Info "Restoring NuGet packages for solution $SolutionPath"
-
-    & $NuGetPath @('restore', $SolutionPath)
-}
-
-
-# Compile task, runs MSBuild to build the solution.
-task Compile  RestorePackages, {
-    Write-Info "Compiling solution $SolutionPath"
-
-    exec {
-        msbuild "$SolutionPath" /nodeReuse:False /target:Build /property:Configuration=Release
+    Push-Location -Path $SourceDir
+    try {
+        exec {
+            dotnet build 

```

Comment:
Do we need this compile step for anything in particular, given these are source packages? Just running `dotnet test` should implicitly do a `dotnet build` for us

Reply:
I thought so, too. But it didn't seem to work reliably. Also, during development I've been finding the speed of being able to build without testing to be useful _(these scripts aren't just for TeamCity!)_. Also, I'm anticipating wanting to publish the results of the build so that the test can be run on another agent (e.g. on Linux). I almost went that far last night, but got tired and stopped.

Reply:
Oh yeah. Now I remember. If you just run `dotnet test` you don't get any useful log output about the build phase. That just seems to magically happen under the covers. Whereas if you run `dotnet build` on its own, you get a lot more log output. I figured that would be useful in a build scenario when things go wrong.

Reply:
Ok, makes sense 👍